Sources/SwiftInspectorKit/FileManager+SwiftFiles.swift

```diff
@@ -38,10 +38,17 @@ extension FileManager {
 
     var swiftFiles: [URL] = []
 
-    let enumerator = self.enumerator(atPath: baseURL.path)
-    while let path = enumerator?.nextObject() as? String {
-      let fileURL = URL(string: path, relativeTo: baseURL)
-      if let fileURL = fileURL, fileURL.pathExtension == "swift" {
```

Comment:
What about this didn't work? In any case, the new approach seems a bit cleaner.

Reply:
Apparently it's a known bug. I discovered this by using this script and then confirmed with the unit test.